README.rst

```diff
@@ -17,6 +17,10 @@ Handles multi-API versions of Azure Storage Data Plane originally from https://g
 
 Change Log
 ----------
+0.3.5
```

Comment:
could we still use 0.3.4 because previous version is invalid?

Reply:
I'm doubting if pypi allow to update the whl only? We can try that.

Reply:
Confirmed with Jianhui, pypi does not support update existing version. So let's just use `0.3.5`.